### PR TITLE
LPS-174732 Improve accessibility entries page

### PR DESCRIPTION
**Pull request by georgel-pop-lr**

### CODE REVIEW

**Comment by georgel-pop-lr** [comment]
ci:test:relevant

**Comment by georgel-pop-lr** [comment]
ci:test:forms

**Comment by georgel-pop-lr** [comment]
ci:test:sf

**Comment by georgel-pop-lr** [comment]
ci:test:relevant

**Comment by georgel-pop-lr** [comment]
ci:test:forms

**Comment by georgel-pop-lr** [comment]
ci:test:forms

**Comment by georgel-pop-lr** [comment]
I don't think that failing test are related to my changes...

**Comment by cgmonte** [comment]
> I think that failing test are related to my changes... Please review.

Hi @georgel-pop-lr I do believe the same. I asked our QA team to confirm.

**Comment by georgel-pop-lr** [comment]
ci:test:sf

**Comment by georgel-pop-lr** [comment]
> > I think that failing test are related to my changes... Please review.
> 
> Hi @georgel-pop-lr I do believe the same. I asked our QA team to confirm.

Hi, I needed to rebase in order to fix conflicts, I will run tests again.

Regards.

**Comment by georgel-pop-lr** [comment]
ci:test:relevant

**Comment by georgel-pop-lr** [comment]
ci:test:forms

**Comment by cgmonte** [comment]
ci:reevaluate:1397401_2899

**Comment by cgmonte** [comment]
ci:reevaluate:1357401_3580

**Comment by georgel-pop-lr** [comment]
Hi all,

Any news regarding this PR?

Regards.

**Comment by georgel-pop-lr** [comment]
ci:test:sf

**Comment by georgel-pop-lr** [comment]
ci:test:relevant

**Comment by georgel-pop-lr** [comment]
ci:test:forms

**Comment by georgel-pop-lr** [comment]
ci:test:sf

**Comment by georgel-pop-lr** [comment]
ci:stop

**Comment by georgel-pop-lr** [comment]
ci:test:forms

**Comment by georgel-pop-lr** [comment]
ci:test:sf

**Comment by georgel-pop-lr** [comment]
ci:test:relevant

**Comment by georgel-pop-lr** [comment]
ci:test:forms

**Comment by georgel-pop-lr** [comment]
ci:test:relevant

**Comment by georgel-pop-lr** [comment]
ci:test:sf

**Comment by georgel-pop-lr** [comment]
Closing this one and sending a new one as this is hard to follow: #2153
